frontend/packages/shipwright-plugin/src/utils.ts

```diff
@@ -1,7 +1,9 @@
-import { K8sResourceKind } from '@console/internal/module/k8s';
+import { IBuild } from '@kubernetes-models/shipwright/shipwright.io/v1alpha1/Build';
+import { IBuildRun } from '@kubernetes-models/shipwright/shipwright.io/v1alpha1/BuildRun';
```

Comment:
Explicit code reduces the risk in the future:

```suggestion
import { IBuild as IBuildV1Alpha1 } from '@kubernetes-models/shipwright/shipwright.io/v1alpha1/Build';
import { IBuildRun as IBuildRunV1Alpha1 } from '@kubernetes-models/shipwright/shipwright.io/v1alpha1/BuildRun';
```